### PR TITLE
Remove user ID prompt

### PR DESCRIPTION
**Pull request by iUnstable0**
its annoying pls xd

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -5,42 +5,42 @@ function createClient() {
...
+    //             userid = _userid.toString();
+    //
+    //             if (fs.existsSync('./userid.txt')) {
+    //                 fs.writeFileSync('./userid.txt', _userid)
+    //             }
+    //
+    //             resolve();
+    //         });
+    //     });
+    // }
```
Comment (RigidStudios) [inline server/index.js]: Just delete the code at this point.

**Review by RigidStudios — CHANGES_REQUESTED** [review]

**Comment by RigidStudios** [comment]
Reason this was included originally was to fulfil setting game activity for when you're outside of studio and playing a game, but I gave up on this project before that ever happened, an implementation of that would be even sweeter but if not I'll merge this once you remove the code.

**Comment by iUnstable0** [comment]
oh, i can do it <3, will work on it tomorrow.

**Comment by iUnstable0** [comment]
i rewrote the backend if that's fine by you xD (you can revert if u dont like it)
I haven't got time to work on the roblox API presence thingy cuz I was out of town, will work on it tmr :)

**Comment by Error-Cezar** [comment]
Since Rigid did not push the changes I did it myself.
UserID prompt is now removed.

**Pull request closed without merging**